tutorials/Tutorial_of_QA_for_FAST.ipynb

```diff
@@ -0,0 +1,337 @@
WHDyzp6JjNbdDWVbq6uamnr0dmnp2peYKv3Ctgum0aa5tcwPhLeUtc96vsr+Sk49cxrdLcY/5QoRNFU0A7OraRfFoMdWd1TyufAxAa1krXxovoKGyAYD3KlxGqZJhd+GVZtMYyh1iZ5cbDarprOEycE89R7JB7A39PpQ7hMUymOsuI9pL2ikZKSFnMgeAtNy0SA8vQNlQWfilqAU+K36W8H06Sl0ig4dVD9nXvi+SNOFjXDu7FMqk2J/Xz0z6zJzXCMudzI38viXu7wA31c5fHOVadyzy3ScTuIuwSVyKXOD/Z+/Oo+PM0vu+fy82YgeIhSBAgvve3Jdh7z09PT1qSRxlbCmyJZ7jNY4dnxxb2ZxzYtk6Vmz/YSV2bJ3YjhwlkRJQVhJFstTSjHqZnulmN8Xmvq8gAGIjAILYdwJv/rj31vtWoQoAQYAAir/POTxVb60vinXfeu+9z30ejrj7Ot1+fKysUZJG6urCwb/HxHd6csdzyR/PZyLDRlkMrRoCbEcGoBhYDzTl9ccGLgGOPDgS+11ehc2Y+CAh5MwPfmZiE43scI8vHS4NZ4tyw65S4uLyAyY+hDbfhGGrxtjwuAp3X45mbuUZKJGBiIiIiIikNc30zMIYY4CTwJYgCH7VGLMBWBsEwddLvGsvRHTq+EZgF0EebAyLiRabcFRnp5vl+XT/p0DyuPw/Pfins77nxq6NsTCYOwkjr1+fgpGauzSXN8duq+yrJG8sHAFuL2unvaydqJ2tOykeKQbs7FJfRh+7WncBdm3A7VnK031xyq5n8tP1E8BOhcTICnDzFFS561eBtiC+dtV41jhTGVPkTtiZnamMKSYyJ2KJB4pGiuJmep4UPaGruCtWLLSpsonHxYmJZuPdWXeH+qp6PrhsVxJ96W6P1tV57e5rnNlxBiCW2CTRg6oH7Gm2xUp3Jrl/nDD2Pz+hMOIZbLstJpzh8dYgkp58GumNxv7W+RmS3a27AbcGFmLraEfd431iIh/+BlDTXRP3mv63+bUU791N/OzSmr41NK5pjHtMVW8VQwnPK0vYjk7m3HDv7WewcrF1xFTUVOZCnZ7Z/Wvsmt1vAb8KDAC/Bxxbyp1abH5afL+JXzB4N3ICdW7rOXbWH4stTn7iLpN1dp7Fmr41DKS4bxIoGA2revTm9zJlpmJJCZJ5++bbFA8X86jUhtr05/VT3VONP636dIZsMP5vLzP2ZGq9u70RxRLL8ldXB+9HQkU2AEMJ4SCFo4VkTmbGQsvGsscozChk3HXxo+tovLM7zs7p/cv77Tuv7VnL5q7N04qHtrq2d9HAYWzHB2yIXUNVQyyxQdSHRz8E7EBG7kQutZFCqG/NsC/RE7O7Nba6z0DuAEceHIkNsoiki+j6WoAm19Z8/boRbFKDbS7pz8OKh3QXdfPdAdtmL7vnZU6GnR6/9jY3UpsPpv8ORsPcyk1YxydIsvbPBCZpNkdf0HRv896kJ6qJNYFSdbxEotTpmd3xIAgOG2MuAQRB0GOMmX4WkGZ2uvSu5UBewgGu1G0fq7f9Pp8++moAtwy88tCOr97YcAOAHW02oncgd2DaDMzbN9+mp6CHgVzbzdnbvJe7wLkUMcJvG7uw2scYN1c2M5trG66ROZUZS5tZPlDO/qb9cSc6qTo8e93f9hC7sLPS3ZeH/YyUQUaWu+i5wVamZ20qHSqlP78/1j5WTaxiMmOSqSy7Xb+2fl7vW95fHuvEgD2JiqYp6DsFH7j29SW2A7TDbe/srWYwbzBpp8e749YFjmbbE7Hqnmo6I2uLTGAITBCbscofy6e7qJvOks7Yc2p67Mj19JWFIivT12592lFjs6f5lNRZ2OyGHW67D/s75tfFTWRNcGvdLd68/SZg1+qN5IzQsdo/I8yuOJXQ4UhVEHidiT/JTJbwpGCsgGj+1TzsMerPIskTpgg7Wn6G1x9LbmkdnjwDremZ3YQxJhN37mCMqYRpCchERERERGSZ0kzP7P4V8PvAGmPMPwF+Dvjlpd2lxfOjSHpLH8q12l1mGzgdwFk3ynPMpY+OxsNfDODNTpeD3830BMY+IVpPx8flFw8XUzxcHCtM9nuEMcUwfYo+Czsb5OuC+DTUM+kptPloVo3baP/X77zOj7FF0FK9D8CuSFjQU6CZcFSsCtvz7TLTnyeynESDTU/vOs1Yzhg13TVkuDGv0axRppiiq2R6kd/n8drd17jlrg9gjyd+lC3DjUL79vUz2FliPw+8EygcKWQu/IyPv5xNwWgB5S6EZ3v7dm5h1wX4/VTYqqxU107ZGRqwGc7AhqEDMJXN8YZwPe4V4ADQUmbnOYdy7coav65uU+em2HqfROXud28Yu0bOt9Qx4n9LDxibPdGnjf/QZXOMyprMYjSybdxrB5E56lWEf5efqfavlKOIC3kG6vTMIgiCOmPMBeA97Hq67wVBcGuWp61IdXWwzR1wknUlNgKjBs67Y9F17ImLj/j1RcN8Ostjjfu5uukq/bk2FW70pGpLx5bY9U7CSs6+w+ND5nII388/FmDLzXeA+EKpPo31tY3Xkv59fqE2kX0G2O2Ktvm/uRt4GtgDeXnktiF3ibvdEIYO6URJlqMSY4sN+u/7m7ff5LO9n1FfPb+Qtbmq7LWBoD5d9G7g4/0f88bV9wEbYjMMvO3u//Doh1T0V3C80ZVAHc9LupZoIeSP5rO/aX9suwHY5K4fMfCDWZKaiCxH/sTfr9nxdau+4cLQL22+xGd7P4slJigdKuWymaKlIj640ycRSZVMpL20nW/1Vsfd5n+XB4EHwHa3D1XYkPA/2/FnKff7ScETaoA97jkF2IQl0fb/RsJz7hLWBsrGFjK9rIKlMgfq9MzCGPMqcCMIgv/ZbRcZY44HQTC3lbwrD4BUCgAAIABJREFUyBYTHkh6sQeTxDoYmUCROzh1BGF8MIQxt35myBczjMYEg4299+oCKDDgb9ls7HsUu+1ibEfsvu9ouctM914ZQFYAhQa+02XLJfY
```

Comment:
Add "If you see the colors for the various sections of the brain are incorrect (white matter is pink or blue) check the order of the inputs for qa_fast_png. The order determines the color-scheme used for each tissue type.